config/webpack.prod.config.js

```diff
@@ -26,9 +26,10 @@ module.exports = Merge.smart(commonConfig, {
         loader: 'babel-loader',
       },
       {
+        type: 'javascript/auto',
```

Comment:
This was the default for everything in webpack 3, we now need to add it explicitly to prevent webpack from "helpfully" doing the wrong thing with json files.